android/src/main/kotlin/com/onegini/mobile/sdk/flutter/OneginiSDK.kt

```diff
@@ -63,7 +64,18 @@ class OneginiSDK {
                 clientBuilder.setConfigModel(`object`)
             }
         } catch (e: Exception) {
-           result.error("10000",e.message, null)
+            val eMessage :String? = e.message
```

Comment:
The kotlin way is to use `?.let { <code here> } ?: <code here>`
Which would result in something like the following (This would be espciall nice with an SdkError constructor which is only 1 line big. 
```kotlin
            e.message?.let { message ->
                SdkError(
                    code = OneWelcomeWrapperErrors.CONFIG_ERROR.code,
                    message = message
                ).flutterError(result)
            } ?: SdkError(
                wrapperError = OneWelcomeWrapperErrors.CONFIG_ERROR
            ).flutterError(result)
```

Reply:
Done